index.js

```diff
@@ -103,6 +104,13 @@ export const startEvaluate = async ({
     })
   }
 
+  const onTransfer = async (to, amount) => {
```

Comment:
Are we okay with making this async, so that we can createPgClient?

Alternatively, instead of passing around the createPgClient here and to public-stats.js, we could just create pgClient in index.js or somewhere else early in the execution, and then just pass pgClient around. One downside here is that we would be initalizing the pgClient even if we don't need to use it, i.e. in onMeasurementsAdded()